packages/theme-check-common/src/liquid-doc/liquidDoc.spec.ts

```diff
@@ -106,7 +106,7 @@ describe('Unit: getSnippetDefinition', () => {
       liquidDoc: {
         examples: [
           {
-            content: '\n          {{ product }}\n',
```

Comment:
We intentionally were capturing this in the past to defer this decision until now since we reckoned that it would be easier to remove this than to add it back.